gensim/models/hdpmodel.py

```diff
@@ -575,9 +570,6 @@ def __init__(self, dictionary=None, topic_data=None, topic_file=None, style=None
 
         self.style = style
 
-    def print_topics(self, num_topics=10, num_words=10):
```

Comment:
since I've already altered the [HdpTopicFormatter](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/hdpmodel.py#L528) class. To unify the model should I remove it's [print_topic](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/hdpmodel.py#L554) method and inherit it from its parent class `basemodel` instead?